packages/js/product-editor/src/components/iframe-editor/iframe-editor.tsx

```diff
@@ -234,9 +248,15 @@ export function IframeEditor( {
 						{ isSidebarOpened && (
 							<div className="woocommerce-iframe-editor__sidebar">
 								<BlockInspector />
+								{ /* For now, when debugging, always render slot */ }
+								{ true && (
+									<ComplementaryArea.Slot scope="woocommerce/product-editor-iframe-editor" />
+								) }
```

Comment:
Just putting the ComplementaryArea.Slot here for now. Eventually, the BlockInspector will also be a fill in this slot, so that it acts like other sidebars.